src/main/twirl/deleted.scala.html

```diff
@@ -7,7 +7,7 @@
         <p>Dear @to.name,
           <br>
           <br>
-          GDPR requires the Guardian to delete API keys that are over 30 months old, unless the user confirms that they wish to keep using their key.</p>
+          GDPR requires the Guardian to delete API keys periodically, unless the user confirms that they wish to keep using their key.</p>
```

Comment:
Definitely more vague is good here.

Reply:
Data protection laws require the Guardian to limit the amount of user data we hold. Unused API keys will be deleted periodically, unless the user confirms that they wish to keep using their key.

(Trying to nudge out GDPR which might not be understood outside of Europe).